packages/gafl-webapp-service/src/session-cache/add-permission.js

```diff
@@ -24,11 +25,13 @@ export default async request => {
   page.permissions.push({})
   status.permissions.push({})
   addressLookup.permissions.push({})
+  analytics.permissions.push({})
```

Comment:
I don't think this is needed

Reply:
Have we decided if this is being changed?

Reply:
Yeah, think it needs to go